### PR TITLE
DOC-2203 feat(DLV2): Add data loading HA doc;

### PR DESCRIPTION
**Pull request by RyanYunboL**
Add data loading HA page.

### CODE REVIEW

File: modules/data-loading/pages/data-loading-ha.adoc [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+== `Setup`
+First, set up a cluster with replication factor greater than 1 following xref:cluster-and-ha-management:ha-cluster.adoc[High Availability Cluster Configuration]
+
+Second, enable Kafka Connect V2 using `gadmin` commands.
+[source, bash]
+----
+gadmin config set KafkaConnect.EnableV2 true
+----
+
+Then run config apply and restart dependency services.
```
Comment (pingxieTG) [inline modules/data-loading/pages/data-loading-ha.adoc]: Let's show the specific command.
And we can use the new config apply --restart-deps option

File: modules/data-loading/pages/data-loading-ha.adoc [hunk 2]
```diff
@@ -0,0 +1,25 @@
+= Data Loading High Availability (HA)
+:page-aliases:
+
+Data Loading High Availability (HA) provides load balancing and automatic fail over in the event of a service disruption.
```
Comment (pingxieTG) [inline modules/data-loading/pages/data-loading-ha.adoc]: Need to give a notice that this is a beta feature and can cause a performance regression.

File: modules/data-loading/pages/data-loading-ha.adoc (hunk 1 above)
Comment (RyanYunboL) [reply modules/data-loading/pages/data-loading-ha.adoc]: Yes. The command is right below. L17-L20

File: modules/data-loading/pages/data-loading-ha.adoc (hunk 2 above)
Comment (RyanYunboL) [reply modules/data-loading/pages/data-loading-ha.adoc]: It no longer causes performance regression. I will update it is a beta version.

**Review by pingxieTG — APPROVED** [review]

File: modules/data-loading/pages/data-loading-ha.adoc [hunk 3]
```diff
@@ -1,8 +1,12 @@
-= Data Loading High Availability (HA)
+= Data Loading High Availability (HA) BETA
 :page-aliases:
 
+NOTE: This is currently a beta feature
+
 Data Loading High Availability (HA) provides load balancing and automatic fail over in the event of a service disruption.
```
Comment (arun-TG-PM) [inline modules/data-loading/pages/data-loading-ha.adoc]: @RyanYunboL @pingxieTG :  Add more description and expand on the benefits and scenarios where Data Loading HA is particularly useful with possible examples. 

Data Loading High Availability (HA) enhances the reliability and efficiency of data loading processes by providing load balancing and automatic failover. This ensures continuous data loading operations even in the event of a service disruption, making it especially valuable for critical applications requiring consistent data availability.

File: modules/data-loading/pages/data-loading-ha.adoc [hunk 4]
```diff
@@ -1,8 +1,12 @@
...
 
+The new data loading mode Data Loading V2 (Beta) provides support for the HA feature.
+
 == `Setup`
 First, set up a cluster with replication factor greater than 1 following xref:cluster-and-ha-management:ha-cluster.adoc[High Availability Cluster Configuration]
```
Comment (arun-TG-PM) [inline modules/data-loading/pages/data-loading-ha.adoc]: @RyanYunboL  Are there any user privileges required to enable Kafka Connect V2? If yes, please explicitly mention it.

**Review by arun-TG-PM — COMMENTED** [review]
@RyanYunboL  - Good work. Quick comment:
Where applicable, include examples that demonstrate the steps involved in enabling and running Data Loading HA. This could include command-line examples or screenshots of the configuration process. Add an example that shows how to verify that Kafka Connect V2 is enabled and how to check the status of Data Loading HA.

File: modules/data-loading/pages/data-loading-ha.adoc (hunk 4 above)
Comment (pingxieTG) [reply modules/data-loading/pages/data-loading-ha.adoc]: No, it is a system config set by `gadmin config set`

**Comment by RyanYunboL** [comment]
> @RyanYunboL - Good work. Quick comment: Where applicable, include examples that demonstrate the steps involved in enabling and running Data Loading HA. This could include command-line examples or screenshots of the configuration process. Add an example that shows how to verify that Kafka Connect V2 is enabled and how to check the status of Data Loading HA.

@arun-TG-PM Enabling data loading V2 is very easy. Simply run the `gadmin` commands as mentioned in the doc. Nothing else is required. I added examples of loading progress and how to validate data loading V2 is enabled. And how to verify it is HA.

**Review by arun-TG-PM — APPROVED** [review]

**Pull request merged**